board.py

```diff
@@ -224,6 +224,19 @@ def player_progress(self, player_id):
         return reached_checkers_num
 
 
+    def check_result (self):
```

Comment:
I think this function should be in `MCTS.py` or `train.py`; board doesnt need to know about rewards of reinforcement learning